### PR TITLE
Update _enterprise-permissions-table.md

### PR DESCRIPTION
**Pull request by BrJan**
Fixed project permissions at the project level. Specifically, the docs show `Analyst` and `Developer` as having `projects_write`, but they only have `projects_read`

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/static/_redirects`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/snippets/_enterprise-permissions-table.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ The project roles enable you to work within the projects in various capacities.
 | Metadata                 |   R   |    R    |       R        |     R     |     R     |     R     |      R      |     R    |                |     R       |     R      |        |
 | Permissions              |   W   |         |       R        |     R     |     R     |           |             |          |                |             |     W      |        |
 | Profile                  |   W   |    R    |       W        |     R     |     R     |     R     |             |          |                |     R       |     R      |        |
-| Projects                 |   W   |    W    |       W        |     W     |     W     |     R     |      R      |          |                |     R       |     W      |        |
+| Projects                 |   W   |    R    |       R        |     R     |     R     |     R     |      R      |          |                |     R       |     R      |        |
```
Comment (aaronbini) [inline website/snippets/_enterprise-permissions-table.md]: in dbt-cloud code, what I see is `Admin`, `Database Admin`, `Git Admin`, and `Team Admin` all have Project write permission

**Comment by mirnawong1** [comment]
hi team, does this PR still need work? or is it good to merge once the conflicts are fixed?

**Comment by mirnawong1** [comment]
hey @BrJan and @aaronbini - wanted to follow up on this pr and see if it's still needed?

**Comment by aaronbini** [comment]
> hey @BrJan and @aaronbini - wanted to follow up on this pr and see if it's still needed?

@mirnawong1 yes we should get this reviewed and merged. I added the lates commit, so I shouldn't really review it at this point, will add my team as reviewers.

**Review by nm0s — APPROVED** [review]
They don't have `project_write` but they do have `project_develop`

**Comment by mirnawong1** [comment]
> They don't have `project_write` but they do have `project_develop`

hey @nm0s thanks so much! so analyst and developer have project develop? does that mean the same thing as 'project read'? or did you mean that comment for Aaron's comment [here](https://github.com/dbt-labs/docs.getdbt.com/pull/4728#discussion_r1448186905)?

**Comment by mirnawong1** [comment]
merging pr per convo[here](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1712854127875899?thread_ts=1712853013.373529&cid=C02NCQ9483C) w @matthewshaver who addressed my [explicit question](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1712853596648079?thread_ts=1712853013.373529&cid=C02NCQ9483C)

**Review by mirnawong1 — APPROVED** [review]
🚢

**Comment by mirnawong1** [comment]
preview is correct: https://docs-getdbt-r7ijy9nbo-dbt-labs.vercel.app/docs/cloud/manage-access/enterprise-permissions#project-permissions-for-project-roles

**Pull request merged**